src/components/OperationTag.js

```diff
@@ -40,18 +33,21 @@ export default class OperationTag extends React.Component {
     let isShownKey = ["operations-tag", tag]
     let showTag = layoutSelectors.isShown(isShownKey, docExpansion === "full" || docExpansion === "list")
 
+    const opBlockSectionKeyId = isShownKey.map(v => escapeDeepLinkPath(v)).join("-")
+    const opBlockSectionCollapseKeyId = opBlockSectionKeyId+'-collapse'
     return (
       <div
-        className={showTag ? "opblock-tag-section is-open" : "opblock-tag-section"}
-      >
-        <h1
-          className={!tagDescription ? "opblock-tag no-desc" : "opblock-tag" }
-          id={isShownKey.map(v => escapeDeepLinkPath(v)).join("-")}
+        className={showTag ? "opblock-tag-section" : "opblock-tag-section"}
+        >
+        <button
+          type="button"
```

Comment:
We can remove this since we're using a button now.

Reply:
Oh we gotta keep that because
> submit: The button submits the form data to the server. This is the default if the attribute is not specified for buttons associated with a `<form>`, or if the attribute is an empty or invalid value.

[reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type)